app/components/ChartSettings/modules/Metadata.js

```diff
@@ -18,16 +18,28 @@ export default class Metadata extends Component {
     title: '',
     caption: '',
     credit: '',
+    subtitle: false,
   };
 
   componentWillMount() {
     this.setState(this.props.metadata);
+    if ('undefined' !== typeof this.props.metadata.subtitle &&
+      false !== this.props.metadata.subtitle) {
```

Comment:
Trying to think if this could yield a false positive for any other falsy values... `null`? `0`? You could do a loose equality check like `false != this.props.metadata.subtitle` , but it looks like 

```
null == false // -> false
0 == false // -> true
```

Maybe check like

```
if ('undefined' !== typeof this.props.metadata.subtitle &&
  ('' === this.props.metadata.subtitle || this.props.metadata.subtitle)
) {
```